<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sekura::WizardItemWidget</class>
 <widget class="QDialog" name="Sekura::WizardItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard Item Edit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pbAdd">
           <property name="toolTip">
            <string>add item</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/011-plus-1.png</normaloff>:/images/011-plus-1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDel">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/003-close.png</normaloff>:/images/003-close.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree">
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Model</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="gb">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lItem">
           <property name="text">
            <string>Type:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbItem">
           <item>
            <property name="text">
             <string>Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Table</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tree</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>List</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TableEdit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Box</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Splitter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Scroll</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tab</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tool</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Spacer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupBox</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Command</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CustomModel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Element</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbMainForm">
           <property name="text">
            <string>Main form</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leID"/>
         </item>
         <item>
          <widget class="QLabel" name="lCaption">
           <property name="text">
            <string>Caption:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leCaption"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="sw">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="columns">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lModel">
               <property name="text">
                <string>Model:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbModel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tw">
             <column>
              <property name="text">
               <string>Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Filter table</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Filter column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Hidden</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="box">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="lDirection">
               <property name="text">
                <string>Direction:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbH">
               <property name="text">
                <string>Horizontal</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbV">
               <property name="text">
                <string>Vertical</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbHV">
               <property name="text">
                <string>Horizontal &amp; Vertical</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>195</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="command"/>
         <widget class="QWidget" name="customModel"/>
         <widget class="QWidget" name="element"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="5"/>
 <tabstops>
  <tabstop>leCaption</tabstop>
  <tabstop>pbAdd</tabstop>
  <tabstop>pbDel</tabstop>
  <tabstop>tree</tabstop>
  <tabstop>cmbItem</tabstop>
  <tabstop>cbMainForm</tabstop>
  <tabstop>cmbModel</tabstop>
  <tabstop>tw</tabstop>
  <tabstop>rbH</tabstop>
  <tabstop>rbV</tabstop>
  <tabstop>rbHV</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
